### PR TITLE
misc: Grafana plugin

### PR DESCRIPTION
**Pull request by manojVivek**
Implements part of #571

### CODE REVIEW

File: ui/packages/app/grafana-datasource-plugin/.github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,75 @@
+name: CI
```
Comment (manojVivek) [inline ui/packages/app/grafana-datasource-plugin/.github/workflows/ci.yml:1]: Please ignore the GitHub workflow files in this child folder, I'll fix these and move them into the root `.github` folder in the next PR.

**Review by brancz — APPROVED** [review]

File: ui/packages/app/grafana-datasource-plugin/src/plugin.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "https://raw.githubusercontent.com/grafana/grafana/master/docs/sources/developers/plugins/plugin.schema.json",
+  "type": "datasource",
+  "name": "Parca Datasource",
```
Comment (metalmatze) [inline ui/packages/app/grafana-datasource-plugin/src/plugin.json:4]:
```suggestion
  "name": "Parca",
```
All the other data sources are simply called by their name only.

File: ui/packages/app/grafana-panel-plugin/src/plugin.json [hunk 3]
```diff
@@ -0,0 +1,35 @@
+{
+  "$schema": "https://raw.githubusercontent.com/grafana/grafana/master/docs/sources/developers/plugins/plugin.schema.json",
+  "type": "panel",
+  "name": "Parca Panel",
```
Comment (metalmatze) [inline ui/packages/app/grafana-panel-plugin/src/plugin.json:4]:
```suggestion
  "name": "Parca Flamegraph",
```
I'd call it "Parca" or simply "Parca Flamegraph".

**Review by metalmatze — COMMENTED** [review]
Amazing work! 
Just renaming a few things for consistency with other panels and datasources.

One thing that I noticed that we should definitely fix is when changing the panel's size. The flamegraph doesn't get updated and looks broken (until you click on "table" and then back to "icicle graph").

**Comment by metalmatze** [comment]
Another thing I just noticed: The Table isn't scrollable right now. 
It'd be great if scrolling the table within the panel would eventually work. :+1:

**Comment by manojVivek** [comment]
Added the resizing and scroll feedback.

**Comment by manojVivek** [comment]
Added this messaging for cors configuration:

Screenshot 2022-10-19 at 11 03 47 AM

Here the origin changes as per the URL from the browser.

**Comment by manojVivek** [comment]
Merging this now, as I need to refer some assets from the GitHub repo URL in the plugin README.md

**Pull request merged**